addon/ember-test-waiters/index.ts

```diff
@@ -1,22 +1,12 @@
-export {
-  WaiterName,
-  Token,
-  Primitive,
-  Waiter,
-  TestWaiter,
-  TestWaiterDebugInfo,
-  PendingWaiterState,
-} from './types';
+import { deprecate } from '@ember/debug';
 
-export {
-  register,
-  unregister,
-  getWaiters,
-  _reset,
-  getPendingWaiterState,
-  hasPendingWaiters,
-} from './waiter-manager';
+deprecate(
+  'Importing from ember-test-waiters is deprecated. Please import from @ember/test-waiters',
+  true,
```

Comment:
https://api.emberjs.com/ember/3.26/functions/@ember/debug/deprecate says:

> `test` **Boolean** A boolean. If falsy, the deprecation will be displayed.

are we sure that this is working as intended? 😅

Reply:
Uggg, I really dislike the deprecate APIs...